lms/djangoapps/course_home_api/course_metadata/v1/tests/test_views.py

```diff
@@ -46,6 +46,13 @@ def test_get_authenticated_user(self):
         # 'Course', 'Wiki', 'Progress' tabs
         assert len(response.data.get('tabs', [])) == 4
 
+    def test_get_authenticated_not_enrolled(self):
+        CourseEnrollment.enroll(self.user, self.course.id)
+        CourseEnrollment.unenroll(self.user, self.course.id)
```

Comment:
I decided to enroll then unenroll the user to account for existing enrollments. This should force us to use the `enrollment.is_active` field.

Reply:
maybe this test should say unenrolled rather than not enrolled then?